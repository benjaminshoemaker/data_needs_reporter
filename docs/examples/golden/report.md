# Data Gaps Report (1 week)

Clusters: 31

## Top 3 Data Gaps (Plain Language)
### Address access_denied, grain_or_type_mismatch, missing_dataset_or_column, performance_limi
- Why: Clustered user questions indicate a recurring data gap to address.
- Root cause: Access denied, Grain or type mismatch, Missing dataset or column, Performance limit, Semantics unclear
- Impact: 37 related items; top audience: Exec
- Examples:
  - [slack] Grant added
  - [slack] Grant added
  - [slack] Grant added
- Datasets:

### Address data gap for stg_orders_039
- Why: Clustered user questions indicate a recurring data gap to address.
- Root cause: N/A
- Impact: 4 related items; top audience: Exec
- Examples:
  - [email] Data freshness for stg_orders_039: Is the join key customer_id or user_id?
  - [email] Data freshness for stg_orders_039: Access request for the marketing schema, permission denied errors. (timeout)
- Datasets:
  - stg_orders_039 (loaded: 2023-01-05T15:50:42Z, SLA: 24h, breach: false)

### Address freshness_breach, missing_dataset_or_column for unspecified dataset
- Why: Clustered user questions indicate a recurring data gap to address.
- Root cause: Freshness breach, Missing dataset or column
- Impact: 3 related items; top audience: Exec
- Examples:
  - [nlq] what is retention by month last last 90 days
  - [nlq] daily gross_margin by plan for eu_only in the last month
  - [slack] Can someone pull cpl by industry for last month? â€” object does not exist
- Datasets:

## Top Backlog Items

- [c_0001] Address access_denied, grain_or_type_mismatch, missing_dataset_or_column, performance_limi
- [c_0031] Address data gap for stg_orders_039
- [c_0003] Address freshness_breach, missing_dataset_or_column for unspecified dataset
- [c_0005] Address access_denied, missing_dataset_or_column, performance_limit for unspecified datase
- [c_0007] Address freshness_breach for unspecified dataset
- [c_0002] Address data gap for mart_finance_027
- [c_0004] Address data gap for stg_products_013
- [c_0006] Address data gap for stg_orders_010
- [c_0008] Address missing_dataset_or_column for unspecified dataset
- [c_0009] Address missing_dataset_or_column for unspecified dataset
- [c_0010] Address data gap for mart_products_023
- [c_0011] Address data gap for mart_products_023
- [c_0012] Address data gap for mart_finance_003
- [c_0013] Address data gap for stg_orders_045
- [c_0014] Address semantics_unclear for unspecified dataset
- [c_0015] Address missing_dataset_or_column for unspecified dataset
- [c_0016] Address data gap for unspecified dataset
- [c_0017] Address missing_dataset_or_column for unspecified dataset
- [c_0018] Address data gap for mart_orders_013
- [c_0019] Address data gap for stg_finance_023, stg_orders_048
- [c_0020] Address grain_or_type_mismatch for unspecified dataset
- [c_0021] Address data gap for mart_orders_012, stg_customers_014
- [c_0022] Address missing_dataset_or_column for unspecified dataset
- [c_0023] Address data gap for mart_marketing_007
- [c_0024] Address data gap for stg_marketing_019
- [c_0025] Address data gap for stg_orders_029
- [c_0026] Address data gap for stg_products_028
- [c_0027] Address data gap for unspecified dataset
- [c_0028] Address data gap for stg_orders_029
- [c_0029] Address data gap for mart_orders_008
- [c_0030] Address data gap for stg_orders_041
