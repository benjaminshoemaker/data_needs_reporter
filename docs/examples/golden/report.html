
<!doctype html>
<meta charset="utf-8"/>
<title>Data Gaps Report</title>
<style>body{font-family:system-ui,Arial;margin:2rem;max-width:900px}li{margin:.25rem 0}</style>
<h1>Data Gaps Report (1 week)</h1>
<p>See backlog.csv for details.</p>

<h2>Top 3 Data Gaps (Plain Language)</h2>
<h3>Address access_denied, grain_or_type_mismatch, missing_dataset_or_column, performance_limi</h3>
<p><strong>Why:</strong> Clustered user questions indicate a recurring data gap to address.</p>
<p><strong>Root cause:</strong> Access denied, Grain or type mismatch, Missing dataset or column, Performance limit, Semantics unclear</p>
<p><strong>Impact:</strong> 37 related items; top audience: Exec</p>
<p><strong>Examples:</strong></p><ul>
<li>[slack] Grant added</li>
<li>[slack] Grant added</li>
<li>[slack] Grant added</li>
</ul>
<p><strong>Datasets:</strong></p><ul>
</ul>
<h3>Address data gap for stg_orders_039</h3>
<p><strong>Why:</strong> Clustered user questions indicate a recurring data gap to address.</p>
<p><strong>Root cause:</strong> N/A</p>
<p><strong>Impact:</strong> 4 related items; top audience: Exec</p>
<p><strong>Examples:</strong></p><ul>
<li>[email] Data freshness for stg_orders_039: Is the join key customer_id or user_id?</li>
<li>[email] Data freshness for stg_orders_039: Access request for the marketing schema, permission denied errors. (timeout)</li>
</ul>
<p><strong>Datasets:</strong></p><ul>
<li>stg_orders_039 (loaded: 2023-01-05T15:50:42Z, SLA: 24h, breach: false)</li>
</ul>
<h3>Address freshness_breach, missing_dataset_or_column for unspecified dataset</h3>
<p><strong>Why:</strong> Clustered user questions indicate a recurring data gap to address.</p>
<p><strong>Root cause:</strong> Freshness breach, Missing dataset or column</p>
<p><strong>Impact:</strong> 3 related items; top audience: Exec</p>
<p><strong>Examples:</strong></p><ul>
<li>[nlq] what is retention by month last last 90 days</li>
<li>[nlq] daily gross_margin by plan for eu_only in the last month</li>
<li>[slack] Can someone pull cpl by industry for last month? â€” object does not exist</li>
</ul>
<p><strong>Datasets:</strong></p><ul>
</ul>
<h2>Top Backlog Items</h2><ul>
<li><code>c_0001</code> Address access_denied, grain_or_type_mismatch, missing_dataset_or_column, performance_limi</li>
<li><code>c_0031</code> Address data gap for stg_orders_039</li>
<li><code>c_0003</code> Address freshness_breach, missing_dataset_or_column for unspecified dataset</li>
<li><code>c_0005</code> Address access_denied, missing_dataset_or_column, performance_limit for unspecified datase</li>
<li><code>c_0007</code> Address freshness_breach for unspecified dataset</li>
<li><code>c_0002</code> Address data gap for mart_finance_027</li>
<li><code>c_0004</code> Address data gap for stg_products_013</li>
<li><code>c_0006</code> Address data gap for stg_orders_010</li>
<li><code>c_0008</code> Address missing_dataset_or_column for unspecified dataset</li>
<li><code>c_0009</code> Address missing_dataset_or_column for unspecified dataset</li>
<li><code>c_0010</code> Address data gap for mart_products_023</li>
<li><code>c_0011</code> Address data gap for mart_products_023</li>
<li><code>c_0012</code> Address data gap for mart_finance_003</li>
<li><code>c_0013</code> Address data gap for stg_orders_045</li>
<li><code>c_0014</code> Address semantics_unclear for unspecified dataset</li>
<li><code>c_0015</code> Address missing_dataset_or_column for unspecified dataset</li>
<li><code>c_0016</code> Address data gap for unspecified dataset</li>
<li><code>c_0017</code> Address missing_dataset_or_column for unspecified dataset</li>
<li><code>c_0018</code> Address data gap for mart_orders_013</li>
<li><code>c_0019</code> Address data gap for stg_finance_023, stg_orders_048</li>
<li><code>c_0020</code> Address grain_or_type_mismatch for unspecified dataset</li>
<li><code>c_0021</code> Address data gap for mart_orders_012, stg_customers_014</li>
<li><code>c_0022</code> Address missing_dataset_or_column for unspecified dataset</li>
<li><code>c_0023</code> Address data gap for mart_marketing_007</li>
<li><code>c_0024</code> Address data gap for stg_marketing_019</li>
<li><code>c_0025</code> Address data gap for stg_orders_029</li>
<li><code>c_0026</code> Address data gap for stg_products_028</li>
<li><code>c_0027</code> Address data gap for unspecified dataset</li>
<li><code>c_0028</code> Address data gap for stg_orders_029</li>
<li><code>c_0029</code> Address data gap for mart_orders_008</li>
<li><code>c_0030</code> Address data gap for stg_orders_041</li>
</ul>
